canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -24,6 +24,11 @@ import ValidationMessage from "./../../components/validation-message";
 import { STATES } from "./../../constants/constants.js";
 import { v4 as uuid4 } from "uuid";
 import { intersection, isEqual } from "lodash";
+import Icon from "./../../../icons/icon.jsx";
```

Comment:
Use the carbon icon directly.  We should stop using icon.jsx.

Reply:
Updated the path.